diff --git a/dist/config/index.js b/dist/config/index.js
index 69e8f80..42d256c 100644
--- a/dist/config/index.js
+++ b/dist/config/index.js
@@ -115,7 +115,7 @@ export function solidStart(options) {
                     define: {
                         "import.meta.env.MANIFEST": `globalThis.MANIFEST`,
                         "import.meta.env.START_SSR": JSON.stringify(start.ssr),
-                        "import.meta.env.START_APP_ENTRY": `"${appEntryPath}"`,
+                        "import.meta.env.START_APP_ENTRY": `"${appEntryPath.replace(/\\/g, "/")}"`,
                         "import.meta.env.START_CLIENT_ENTRY": `"${handlers.client}"`,
                         "import.meta.env.START_DEV_OVERLAY": JSON.stringify(start.devOverlay)
                     },
@@ -163,19 +163,19 @@ export function solidStart(options) {
                 {
                     envConsumer: "client",
                     envName: VITE_ENVIRONMENTS.client,
-                    getRuntimeCode: () => `import { createServerReference } from "${normalize(fileURLToPath(new URL("../server/server-runtime", import.meta.url)))}"`,
+                    getRuntimeCode: () => `import { createServerReference } from "${normalize(fileURLToPath(new URL("../server/server-runtime", import.meta.url))).replace(/\\/g, "/")}"`,
                     replacer: opts => `createServerReference('${opts.functionId}')`
                 },
                 {
                     envConsumer: "server",
                     envName: VITE_ENVIRONMENTS.server,
-                    getRuntimeCode: () => `import { createServerReference } from '${normalize(fileURLToPath(new URL("../server/server-fns-runtime", import.meta.url)))}'`,
+                    getRuntimeCode: () => `import { createServerReference } from '${normalize(fileURLToPath(new URL("../server/server-fns-runtime", import.meta.url))).replace(/\\/g, "/")}'`,
                     replacer: opts => `createServerReference(${opts.fn}, '${opts.functionId}')`
                 }
             ],
             provider: {
                 envName: VITE_ENVIRONMENTS.server,
-                getRuntimeCode: () => `import { createServerReference } from '${normalize(fileURLToPath(new URL("../server/server-fns-runtime", import.meta.url)))}'`,
+                getRuntimeCode: () => `import { createServerReference } from '${normalize(fileURLToPath(new URL("../server/server-fns-runtime", import.meta.url))).replace(/\\/g, "/")}'`,
                 replacer: opts => `createServerReference(${opts.fn}, '${opts.functionId}')`
             }
         }),
